src/main/java/moriyashiine/aylyth/mixin/MilkBucketItemMixin.java

```diff
@@ -0,0 +1,33 @@
+package moriyashiine.aylyth.mixin;
+
+import moriyashiine.aylyth.common.registry.ModPotions;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.effect.StatusEffect;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.MilkBucketItem;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(MilkBucketItem.class)
+public class MilkBucketItemMixin {
+    @Unique
+    private static final StatusEffect[] persistentStatusEffects = { ModPotions.CRIMSON_CURSE_EFFECT };
+
+    @Inject(method = "finishUsing", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/LivingEntity;clearStatusEffects()Z"), cancellable = true)
```

Comment:
Please rewrite this as a WrapOperation, collecting all of the persistent effects from the tag mentioned before and then calling the original operation (which performs the clearing and applying other wrapped operations from other mods) then reapply all of the persistent effects.